src/metabase/driver/common/parameters/values.clj

```diff
@@ -48,7 +48,7 @@
     :display-name                 su/NonBlankString
     :type                         ParamType
     (s/optional-key :dimension)   [s/Any]
-    (s/optional-key :card)        su/IntGreaterThanZero
+    (s/optional-key :card-id)     su/IntGreaterThanZero
```

Comment:
I see the FE uses `card_id` and the BE uses `card-id`. Is that automatically converted somewhere?

Reply:
It is! Though, I'm not sure where that happens.